python/pyspark/sql/connect/dataframe.py

```diff
@@ -1775,6 +1775,12 @@ def _to_table(self) -> Tuple["pa.Table", Optional[StructType]]:
         assert table is not None
         return (table, schema)
 
+    def _toArrow(self) -> "pa.Table":
+        table = self._to_table()[0]
+        return table
+
+    _toArrow.__doc__ = PySparkDataFrame._toArrow.__doc__
```

Comment:
@HyukjinKwon is this line no longer needed after https://github.com/apache/spark/pull/46129?